UserManagement/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Kindly add all folders and files which are not required for the project to .gitingnore file.

Reply:
Will do it. Thanks